doc/productivity.md

```diff
@@ -32,6 +33,17 @@ Install `ccache` through your distribution's package manager, and run `./configu
 
 To use ccache for all your C/C++ projects, follow the symlinks method [here](https://ccache.samba.org/manual/latest.html#_run_modes) to set it up.
 
+To get the most out of ccache, put something like this in `~/.ccache/ccache.conf`:
+
+```
+max_size = 50.0G  # or whatever cache size you prefer; default is 5G; 0 means unlimited
```

Comment:
I'd rather not bikeshed, but is there any reason to increase this over the default? I never touched this and it seems to work fine. Increasing it too much means you'd have files laying around that are outdated at some point anyway, because your distributions compiler got updated in the meantime.

I guess it could be relevant when you compile more than one project regularly?

Reply:
FWIW I don't compile anything other than different branches of bitcoin core, and I've only been using it a couple months, and my cache is currently 12.5 GB. I increased it because even just testing it on 2-3 branches of core, I was seeing it start to fill significantly. I think 5 GB is just a ludicrously puny size in the modern era of storage. I feel pretty happy about 50 currently, and you're right that there are decreasing marginal benefits at some point (but I think 5 GB is WAY below that point, and a really stupid default.)